digitizer/forms.py

```diff
@@ -81,35 +85,31 @@ def __init__(self, plot, tabs):  # pylint: disable=redefined-outer-name
         self.btn_plot = pn.widgets.Button(name='Plot', button_type='primary')
         self.btn_plot.on_click(self.on_click_plot)
 
+        self.btn_xferjson = pn.widgets.Button(name='Transfer data from JSON',
```

Comment:
Another option is to use the "prefill" button for this.
Put the JSON upload next to his button. The prefill callback will check whether a file has been uploaded. If not, it uses sample data (by the way, ideally "prefill" would parse a sample JSON, which would then also become a test in CI).

Reply:
excellent idea. I have a handful of test JSON files that can be co-opted for the tests:
1. basic single component
2. single component in log-scale pressure
3. full quaternary multicomponent (all compositions specified, isotherms for all species)
4. "partial" quaternary multicomponent (four species specified, but isotherm for only one species included [only has composition for that species]) - odd case, but we find isotherms like this.